web/src/components/profile/ProfileField.tsx

```diff
@@ -16,28 +19,32 @@ interface Props {
 }
 
 export const ProfileField = (props: Props): ReactElement => {
+  const { isFormFieldInValid } = useFormContextFieldHelpers(props.fieldName, profileFormContext);
+
   if (props.isVisible === false) {
     return <></>;
   }
   return (
     <>
-      <div className="margin-y-4" id={`question-${props.fieldName}`}>
-        {props.locked ? (
-          <LockedProfileField fieldName={props.fieldName} valueFormatter={props.lockedValueFormatter} />
-        ) : (
-          <>
-            {!props.noLabel && (
-              <FieldLabelProfile
-                fieldName={props.fieldName}
-                isAltDescriptionDisplayed={props.displayAltDescription}
-                hideHeader={props.hideHeader}
-                boldAltDescription={props.boldAltDescription}
-              />
-            )}
-            <div className={props.noLabel ? "margin-bottom-05" : ""}>{props.children}</div>
-          </>
-        )}
-      </div>
+      <WithErrorBar hasError={isFormFieldInValid} type={"ALWAYS"}>
```

Comment:
[dust] Should this be wrapped at this level? It's not like the locked field could ever have an error, right. Maybe it should be wrapped at line 35 instead?